app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+class PostController extends Controller
+{
+    /**
+     * Show the list of blog posts.
+     */
+    public function index()
+    {
+        $posts = [
+            ['title' => 'Fieldwork', 'slug' => 'fieldwork', 'description' => "description"],
+            ['title' => 'First Feedback', 'slug' => 'firstfeedback', 'description' => "After two weeks of working on my website I had to pitch it to my teachers and fellow student. In this blog i'll show what feedback i got on my website."],
```

Comment:
Remember to follow up on these automatically generated PHPcs comments. In the exam, there might be a point deduction when we find them in your code.